# HG changeset patch
# User Rocco Rutte <pdmef@gmx.net>
# Date 1212156897 -7200
# Node ID 2f59be26ad22e443da85f862584f6a8fe84ce387
# Parent bf74c85070c4ea755664d4ec87c6eb3fff3f6da5
Define prototypes for compatibility functions in protos.h to silence compiler warnings

--- a/protos.h	Thu May 29 14:18:25 2008 +0200
+++ b/protos.h	Fri May 30 16:14:57 2008 +0200
@@ -523,4 +523,26 @@
 void ci_bounce_message (HEADER *, int *);
 int ci_send_message (int, HEADER *, char *, CONTEXT *, HEADER *);
 
-
+/* prototypes for compatibility functions */
+
+#ifndef HAVE_SETENV
+int setenv (const char *, const char *, int);
+#endif
+
+#ifndef HAVE_STRCASECMP
+int strcasecmp (char *, char *);
+int strncasecmp (char *, char *, size_t);
+#endif
+
+#ifndef HAVE_STRDUP
+char *strdup (const char *);
+#endif
+
+#ifndef HAVE_STRSEP
+char *strsep (char **, const char *);
+#endif
+
+#ifndef HAVE_STRTOK_R
+char *strtok_r (char *, const char *, char **)
+#endif
+

